input/definitions/MeasureReport/StructureDefinition-criteriaReference.json

```diff
@@ -0,0 +1,88 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "criteriaReference",
+  "url": "http://hl7.org/fhir/StructureDefinition/criteriaReference",
+  "version": "1.0.0",
+  "name": "CriteriaReferenceExtension",
+  "title": "Criteria Reference Extension",
+  "status": "active",
+  "experimental": false,
+  "date": "2023-12-15",
+  "publisher": "HL7 International / Clinical Decision Support",
+  "contact": [
+    {
+      "telecom": [
+        {
+          "system": "url",
+          "value": "http://www.hl7.org/Special/committees/dss"
+        }
+      ]
+    }
+  ],
+  "description": "Specifies which criteria is the input for calculations.",
+  "jurisdiction": [
+    {
+      "coding": [
+        {
+          "system": "urn:iso:std:iso:3166",
+          "code": "US"
+        }
+      ]
+    }
+  ],
+  "purpose": "The definition must be able to specify which criteria is the input used.",
```

Comment:
Should be:
"Evaluated resources and supplemental data included in the measure report should be able to identify which criteria were responsible for their inclusion in the report. For example, a MedicationRequest reference can point to the denominator criteria to indicate it was considered as part of the evaluation of that criteria"

Reply:
```suggestion
  "purpose": "In cases where multiple populations of a specific type exist (such as continuous variable and ratio measures), measures must be able to identify which specific criteria is intended as the input. In addition, evaluated resources and supplemental data included in the measure report should be able to identify which criteria were responsible for their inclusion in the report. For example, a MedicationRequest reference can point to the denominator criteria to indicate it was considered as part of the evaluation of that criteria.",
```